static/app/components/forms/controls/rangeSlider/sliderProps.tsx

```diff
@@ -0,0 +1,68 @@
+type SliderProps = {
```

Comment:
I would name this file `types`

Reply:
@evanpurkhiser we usually colocate components and their prop definitions though, I think this should be a part of slider.tsx and exported there